micropython/micropython-1.15-patchset/mpy-cross.patch

```diff
@@ -1,6 +1,33 @@
+diff -Naur micropython-1.15/mpy-cross/main.c micropython-phoenix/mpy-cross/main.c
+--- micropython-1.15/mpy-cross/main.c	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/mpy-cross/main.c	2021-05-21 10:42:06.561325533 +0200
+@@ -339,7 +339,7 @@
+     return main_(argc, argv);
+ }
+ 
+-uint mp_import_stat(const char *path) {
++mp_import_stat_t mp_import_stat(const char *path) {
+     (void)path;
+     return MP_IMPORT_STAT_NO_EXIST;
+ }
+diff -Naur micropython-1.15/mpy-cross/Makefile micropython-phoenix/mpy-cross/Makefile
+--- micropython-1.15/mpy-cross/Makefile	2021-04-18 16:27:07.000000000 +0200
++++ micropython-phoenix/mpy-cross/Makefile	2021-05-21 10:49:35.737163519 +0200
+@@ -17,9 +17,9 @@
+ INC += -I$(TOP)
+ 
+ # compiler settings
+-CWARN = -Wall -Werror
++CWARN = -Wall
+ CWARN += -Wextra -Wno-unused-parameter -Wpointer-arith
+-CFLAGS = $(INC) $(CWARN) -std=gnu99 $(CFLAGS_MOD) $(COPT) $(CFLAGS_EXTRA)
++CFLAGS += $(INC) $(CWARN) -std=gnu99 $(CFLAGS_MOD) $(COPT) $(CFLAGS_EXTRA)
```

Comment:
why do we need to change it to `+=` ? Seems like You're trying to simultaneously use ports-provided `CFLAGS` and `CFLAGS_EXTRA`